service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/client/RestClient.java

```diff
@@ -16,18 +20,25 @@
  * Would perform HTTP GET against http://localhost:8080/organization/1234567
  * and return the HTTP Response body as a String
  */
+@Component
 public class RestClient {
+
+    private final IntegrasjonspunktProperties props;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "props" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)